pkg/osquery/tables/secedit/secedit.go

```diff
@@ -132,7 +133,7 @@ func (t *Table) execSecedit(ctx context.Context, mergedPolicy bool) ([]byte, err
 	rd := transform.NewReader(file, unicode.UTF16(unicode.LittleEndian, unicode.UseBOM).NewDecoder())
 	data, err := ioutil.ReadAll(rd)
 	if err != nil {
-		return nil, errors.Wrapf(err, "error reading secedit output file: %s", err)
+		return nil, fmt.Errorf("error reading secedit output file: %s: %w", err, err)
```

Comment:
Is something else supposed to be for `%s` instead of including `err` twice? Maybe `dst` like on 127 above?

Reply:
fixed